lib/defaults.js

```diff
@@ -58,6 +58,11 @@ module.exports = function (inflect) {
   inflect.irregular('move', 'moves');
   inflect.irregular('cow', 'kine');
   inflect.irregular('zombie', 'zombies');
+  inflect.irregular('oaf', 'oafs', true);
+  inflect.irregular('jefe', 'jefes');
+  inflect.irregular('save', 'saves');
+  inflect.irregular('safe', 'safes');
+  inflect.irregular('fife', 'fifes');
 
-  inflect.uncountable(['equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep', 'jeans']);
+  inflect.uncountable(['equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep', 'jeans', 'sushi']);
```

Comment:
Also slipping this in, see http://www.theatlantic.com/technology/archive/2016/01/whats-the-plural-of-emoji-emojis/422763/ for discussion of noncountability of sushi